kubeflow/core/tensorboard.libsonnet

```diff
@@ -0,0 +1,82 @@
+{
+	parts(namespace, name,):: {
+		service:: {
+		  "apiVersion": "v1", 
+		  "kind": "Service", 
+		  "metadata": {
```

Comment:
Can you add an annotation to add an Ambassador route as in
https://github.com/jlewi/kubeflow-rl/blob/master/rl-app/components/tensorboard.libsonnet#L9